.github/workflows/system-test.yml

```diff
@@ -0,0 +1,52 @@
+name: system-test
+
+on:
+  pull_request: ~
```

Comment:
We have to remove this. Otherwise, this will always fail on fork PRs. (Because they don't have access to to secrets)

```suggestion
```

Reply:
I was think wrongly. We don't have access to secrets in a workflow from a forked PRs, not matter what.

We can only run this on main for now.

@marclop WDYT?

Reply:
https://github.com/elastic/apm-queue/pull/170/commits/2fde13409a8d39b14b510e3ea652d33f39d60b63 should fix the problem.

A workflow with the `pull_request_target` trigger always runs the workflow that is checked-in in the `main` branch.

Using the `is-pr-approved` action, we can check the conditions in [Motivation & Context]( https://github.com/elastic/apm-pipeline-library/tree/main/.github/actions/is-pr-approved#motivation--context).

only then the workflow is continued.

Reply:
Awesome!